CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Gluecodium project Release Notes
 
+## Unreleased
+### Bug fixes:
```

Comment:
I would consider this an improvement, a.k.a feature

Reply:
Done